app/main/views/search.py

```diff
@@ -20,62 +22,135 @@
+            capabilities.append(option)
+            capabilities_plain.append(role['category'])
+
     filters = [
-        Filter('Capabilities', [
-            Option('category', 'pm', 'Product Management', False),
-            Option('category', 'ba', 'Business Analysis', False),
-            Option('category', 'dm', 'Delivery Management and Agile Coaching', False),
-            Option('category', 'ur', 'User Research', False),
-            Option('category', 'sd', 'Service Design and Interaction Design', False),
-            Option('category', 'sd', 'Service Design and Interaction Design', False),
-            Option('category', 'perf', 'Performance and Web Analytics', False),
-            Option('category', 'acc', 'Inclusive Design and Accessibility', False),
-            Option('category', 'dta', 'Digital Transformation Advisors', False),
-        ]),
-
-        # Roles are deprecated
-        #        Filter('Roles', [
-        #            Option('role', 'tl', 'Technical Lead', False),
-        #            Option('role', 'dev', 'Developer', False),
-        #            Option('role', 'hck', 'Ethical Hacker', False),
-        #            Option('role', 'ops', 'Web Devops Engineer', False),
-        #        ])
-
-    ]
-    results = [
-        Result(
-            'PublicNet',
-            'PublicNet is a wholly owned Indigenous non-profit company dedicated to'
-            'providing specialised Business Analysis services for the public and community sector.',
+        Filter('Capabilities', capabilities),
+        ]
+
+    #    results = [
+    #        Result(
+    #            'PublicNet',
+    #            'PublicNet is a wholly owned Indigenous non-profit company dedicated to'
+    #            'providing specialised Business Analysis services for the public and community sector.',
+    #            [Badge('badge-security', 'Security clearance'), Badge('badge-tick', 'ABC compliant')],
+    #            [Category('Business Analysis'), Category('User Research')],
+    #            [ExtraDetail('Location', 'Sydney'),
+    #             ExtraDetail('Able to work interstate', 'Yes'),
+    #             ExtraDetail('Indigenous owned', 'Yes'),
+    #             ExtraDetail('Employs Indigenous Australians', 'Yes'),
+    #             ExtraDetail('Uses Indigenous Providers', 'Yes'),
+    #             ExtraDetail('Worked with government before', 'Yes'),
+    #             ExtraDetail('Size of company', '20 people')],
+    #            '/'
+    #        ),
+    #        Result(
+    #            'VX Productions',
+    #            'VX is a leading provider of web design and hosting solutions.',
+    #            [Badge('badge-tick', 'XYZ compliant')],
+    #            [Category('Delivery Management and Agile Coaching')],
+    #            [ExtraDetail('Location', 'Sydney'), ExtraDetail('Able to work interstate', 'Yes')],
+    #            '/'
+    #        ),
+    #    ]
+
+    must = []
+    for cat in cat_list:
```

Comment:
Please write this in a more readable way.  I had no idea what "must" meant.  Reading the code I figured out that it's a category query, but just calling it that would help people.  Similarly for "cat_list".  The total time taken for your colleagues to figure out this is a "category_list" is small, but probably longer than the time taken to type those 5 more letters.